1.3/Defs/chitchat_thoughts.xml

```diff
@@ -958,6 +966,10 @@
 				<!-- ===Good opinion of recipient=== -->
 					<li>r_logentry(INITIATOR_opinion>=0)->[good_opinion]</li>
 					<li>good_opinion->You're right, [recipient]. We should be more careful to avoid such things.</li>
+					<li>good_opinion->Oh, no... Sorry, [recipient]. We'll all try to be more careful.</li>
+					<li>good_opinion->We'll do our best to tread softly, [recipient].</li>
+					<li>good_opinion(INITIATOR_trait==abrasive)->Not my fault, [RECIPIENT_nameDef]. Sucks to be you, I guess. Sorry.</li>
+					<li>good_opinion(INITIATOR_trait==kind)->Poor [RECIPIENT_nameDef]! Take it easy today, and be sure to get a good night's sleep tonight.</li>
```

Comment:
Pawns might sleep during daytime... maybe it's best to be more generic?... "be sure you get a full rest next time you sleep!"

Reply:
👍 Changing! Good catch!